playtron-os-files/playtron-os-files.spec

```diff
@@ -1,10 +1,10 @@
-Name: playtron-os-scripts
-Version: 0.5.1
+Name: playtron-os-files
+Version: 0.5.2
 Release: 1%{?dist}
 Summary: Scripts and services for a gaming OS
 License: Apache-2.0
-URL: https://github.com/playtron-os/playtron-os-scripts
-Source0: https://github.com/playtron-os/playtron-os-scripts/archive/refs/tags/%{version}.tar.gz
+URL: https://github.com/playtron-os/playtron-os-files
+Source0: https://github.com/playtron-os/playtron-os-files/archive/refs/tags/%{version}.tar.gz
 BuildArch: noarch
 Requires: cloud-utils-growpart fio parted
```

Comment:
I am not sure off the top of my head what is needed for the RPM spec, but we need to specify that `playtron-os-files` is a replacement to `playtron-os-scripts`. That should help out with upgrades.

Reply:
I don't think anything depends on it?

Reply:
We can add a "conflicts", but I don't think it makes too much sense since `playtron-os-scripts` will no longer be distributed.

Reply:
For Playtron OS updates, this will work as-is (since it knows that `playtron-os-scripts` is removed and `playtron-os-files` should be installed). I know a few people on the team are using Fedora Workstation with some RPMs of ours manually installed. For them, the upgrade path is not as clear or automatic.

In Arch Linux PKGBUILDs, this is handled by adding `conflicts` and `replaces`. The  Fedora Project recommendation appears to be to use `Obsoletes:`. `Conflicts` is highly discouraged.

All we need to add here is:

```
Obsoletes: playtron-os-scripts
```

https://fedoraproject.org/wiki/Upgrade_paths_%E2%80%94_renaming_or_splitting_packages
https://fedoraproject.org/wiki/Packaging:Conflicts